### PR TITLE
chore(application-system): Clean up old children residence change application

### PR DESCRIPTION
**Pull request by norda-gunni**
# ...

Attach a [link to issue](https://app.asana.com/0/1207402031871459/1208647192325484/f) if relevant

## What

Remove the old children residence change application and functionality that is used exclusively by it.

## Why

The old application has been replaced by a newer version and is essentially just getting in the way

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/application-system/api/src/app/modules/application/application.controller.ts [hunk 1]
```diff
@@ -1151,8 +967,6 @@ export class ApplicationController {
     // delete the entry in Payment table to prevent FK error
     await this.paymentService.delete(existingApplication.id, user)
 
-    await this.fileService.deleteAttachmentsForApplication(existingApplication)
```
Comment (Toti91) [inline apps/application-system/api/src/app/modules/application/application.controller.ts:1154]: Should this be removed? Isn't this for pruning all attachments e.g. file-uploads?

File: apps/application-system/api/src/app/modules/application/application.controller.ts [hunk 2]
```diff
@@ -891,182 +883,6 @@ export class ApplicationController {
...
-    @Param('attachmentKey') attachmentKey: string,
-    @CurrentUser() user: User,
-  ): Promise<PresignedUrlResponseDto> {
-    const existingApplication =
-      await this.applicationAccessService.findOneByIdAndNationalId(id, user)
-
-    if (!existingApplication.attachments) {
-      throw new NotFoundException('Attachments not found')
-    }
-
```
Comment (Toti91) [inline apps/application-system/api/src/app/modules/application/application.controller.ts:1060]: Are these methods used for normal attachments also?

**Review by Toti91 — COMMENTED** [review]
Two questions

File: apps/application-system/api/src/app/modules/application/application.controller.ts (hunk 1 above)
Comment (norda-gunni) [reply apps/application-system/api/src/app/modules/application/application.controller.ts:1154]: Great catch, thanks. That was meant to stay.

**Comment by Toti91** [comment]
Wait now I'm confused by this commit, do we need to add all the dokobit signing stuff back in? [029f75e](https://github.com/island-is/island.is/pull/16735/commits/029f75ec18dcb190221bdab0cecd006db743f2c4)

File: apps/application-system/api/src/app/modules/application/application.controller.ts (hunk 2 above)
Comment (norda-gunni) [reply apps/application-system/api/src/app/modules/application/application.controller.ts]: Re-added those

**Comment by norda-gunni** [comment]
Because judicial was using it?

**Comment by norda-gunni** [comment]
I re-added some stuff that I was a bit overzealous in removing

**Review by Toti91 — APPROVED** [review]
LGTM

**Pull request merged**